backend/test/integration/healthz_api_test.go

```diff
@@ -26,8 +26,9 @@ import (
 
 type HealthzApiTest struct {
 	suite.Suite
-	namespace     string
-	healthzClient *api_server.HealthzClient
+	namespace         string
+	resourceNamespace string
```

Comment:
I don't have the full context, so to confirm, is it being used somewhere during the test?

Reply:
See the comment on namespace vs. resourceNamespace below. The latter isn't used in this test, but all other tests has are styled the same way so I thought it might be good for consistency sake to include resourceNamespace.